docs/guide/yaml/example_yaml/test-app-with-enrichers-slightly-simpler.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# example showing how enrichers can be set 
```

Comment:
a refactoring of these files would be nice.  let's wait until we have a massively simpler syntax however.